client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -98,45 +99,110 @@ class PluginsBrowserListElement extends Component {
 	renderPlaceholder() {
 		/* eslint-disable wpcalypso/jsx-classname-namespace */
 		return (
-			<li className="plugins-browser-item is-placeholder">
+			<Card className="plugins-browser-item is-placeholder">
 				<span className="plugins-browser-item__link">
 					<div className="plugins-browser-item__info">
+						<div className="plugins-browser-item__title-wrapper">
+							<div className="plugins-browser-item__title">…</div>
+							<div className="plugins-browser-item__description">…</div>
+							<div className="plugins-browser-item__author">…</div>
+						</div>
 						<PluginIcon size={ this.props.iconSize } isPlaceholder={ true } />
-						<div className="plugins-browser-item__title">…</div>
-						<div className="plugins-browser-item__author">…</div>
 					</div>
-					<Rating rating={ 0 } size={ 12 } />
+					<div className="plugins-browser-item__meta is-placeholder">
+						<div className="plugins-browser-item__ratings">
+							<Rating rating={ 0 } size={ 16 } />
+						</div>
+					</div>
 				</span>
-			</li>
+			</Card>
 		);
 		/* eslint-enable wpcalypso/jsx-classname-namespace */
 	}
 
+	renderLastUpdated = () => {
+		const { plugin, moment } = this.props;
+
+		if ( plugin && plugin.last_updated ) {
+			const dateFromNow = moment.utc( plugin.last_updated, 'YYYY-MM-DD hh:mma' ).fromNow();
+
+			return (
+				<div className="plugins-browser-item__last-updated">
+					{ this.props.translate( 'Updated %(dateFromNow)s', { args: { dateFromNow } } ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'Last updated %(dateFromNow)s' )` *ES Score: 8*